Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build build-stub test test-vips lint generate sqlc-generate openapi setup-hooks run clean
+.PHONY: build build-stub docker test test-vips lint generate sqlc-generate openapi setup-hooks run clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)